macros/get_message_columns.sql

```diff
@@ -19,17 +19,17 @@
     {"name": "updated_at", "datatype": dbt.type_timestamp()}
 ] %}
 
+{% if target.type == 'snowflake' %} 
+    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "message_from"}) }} 
+{% else %} 
+    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "message_from"}) }} 
+{% endif %}
+ 
 {% if target.type == 'snowflake' %}
-    {{ columns.append({"name": "FROM", "datatype": dbt.type_string(), "quote": True, "alias": "message_from"}) }}
-{% else %}
-    {{ columns.append({"name": "from", "datatype": dbt.type_string(), "quote": True, "alias": "message_from"}) }}
-{% endif %},
-
-{% if target.type == 'snowflake' %}
-    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "message_to"}) }}
-{% else %}
-    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "message_to"}) }}
-{% endif %},
+    {{ columns.append({"name": "TO", "datatype": dbt.type_string(), "quote": True, "alias": "message_to"}) }} 
+{% else %} 
+    {{ columns.append({"name": "to", "datatype": dbt.type_string(), "quote": True, "alias": "message_to"}) }} 
+{% endif %}
```

Comment:
Same comment from previous macro. Is there a reason we do not combine these conditionals? Also, be sure to remove the trailing comma.

Reply:
done!

Reply:
Same comment as the previous